File: dotnet-backend/CanchasSinteticas.Infrastructure/Data/Models/NoShowModel.cs
using System.ComponentModel.DataAnnotations;

namespace CanchasSinteticas.Infrastructure.Data.Models;

public class NoShowModel
{
    [Key] public string Id { get; set; } = default!;
    public string ReservationId { get; set; } = default!;
    public string UserId { get; set; } = default!;
    public string RecordedAt { get; set; } = default!;
}

File: backend/infrastructure/models/no_show_model.py
from sqlalchemy import Column, ForeignKey, Integer, String

from infrastructure.database import Base


class NoShowModel(Base):
    __tablename__ = "no_shows"

    id = Column(Integer, primary_key=True, autoincrement=True)
    reservation_id = Column(String, ForeignKey("reservations.id"), nullable=False)
    user_id = Column(String, nullable=False)
    cancelled_at = Column(String, nullable=False)
